src/redux/sagas/transactionsSaga.ts

```diff
@@ -102,8 +103,8 @@ export function* fetchAndUpdateTxs(
 function* updateAssets({ payload }: ReturnType<typeof setTransaction>) {
   if (payload?.vout) {
     for (const out of payload.vout) {
-      if (!isBlindedOutputInterface(out)) {
-        yield put(addAsset(out.asset));
+      if (isUnblindedOutput(out)) {
+        yield put(addAsset(out.unblindData.asset.toString('hex')));
```

Comment:
One of the main change of the LDK types refacto is the way we access asset hash (as string) and sats (as number), see [here](https://github.com/vulpemventures/ldk/pull/92).
> Another main change is the way we access Output values and assets: getSats takes an Output and returns the number of satoshis (throw an error if not possible due to blinding). Same for the asset with getAsset.

so basically instead of `out.unblindData.asset` you MUST use `getAsset` function from LDK to access it. getAsset encapsulates all the logic of "is it blinded or unblinded ? should I reverse ? remove the first byte etc...).